force-app/emails/lwc/emailDeliveryReport/emailDeliveryReport.html

```diff
@@ -0,0 +1,23 @@
+<template>
+    <lightning-quick-action-panel header={label.emailDeliveryReportHeading}>
+        <h3 class="slds-text-heading_small slds-m-bottom_medium">
+            {label.emailNotSentHeading}
+        </h3>
+
+        <lightning-datatable
+            data={errors}
+            columns={columns}
+            lwc:if={hasErrors}
+            key-field="id"
+            hide-checkbox-column>
+        </lightning-datatable>
+
```

Comment:
Could we also include a message if any messages were successfully sent? Maybe just a summary: "1 message was successfully sent."

Reply:
Also, will some emails be sent if others fail? I didn't see the one I expected come through.

Reply:
Nevermind, I hadn't verified my email. (Is that a condition we can detect/warn about?) Note to self: Let's make sure to include a note about verifying email in documentation.

Reply:
I added functionality which now checks the "verified" status of the user's own email before allowing them to choose it as an originator option.  These changes are included in 841f87e.  Review that commit (and commit message) for more details.